content/en/docs/concepts/extend-kubernetes/service-catalog.md

```diff
@@ -32,7 +32,7 @@ The application can access the message queue as a service.
 
 Service Catalog uses the [Open service broker API](https://github.com/openservicebrokerapi/servicebroker) to communicate with service brokers, acting as an intermediary for the Kubernetes API Server to negotiate the initial provisioning and retrieve the credentials necessary for the application to use a managed service.
 
-It is implemented as an extension API server and a controller, using etcd for storage. It also uses the [aggregation layer](/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/) available in Kubernetes 1.7+ to present its API.
+It is implemented as an extension API server and a controller, using etcd for storage.
```

Comment:
This implies that there's no CRD in use, just an APIService. That doesn't match the PR description, though. I'm puzzled.

Reply:
@sftim I can see how this can be confusing to readers. I have reworded this sentence to now reflect the description used in the Service Catalog docs: looks simpler to read.